scripts/watch.mjs

```diff
@@ -180,13 +177,20 @@ const setupBuiltinExtensionApiWatcher = name => {
     });
 
     await viteDevServer.listen();
-    setupBuiltinExtensionApiWatcher('compose');
-    setupBuiltinExtensionApiWatcher('docker');
-    setupBuiltinExtensionApiWatcher('kube-context');
-    setupBuiltinExtensionApiWatcher('lima');
-    setupBuiltinExtensionApiWatcher('podman');
-    setupBuiltinExtensionApiWatcher('kind');
-    setupBuiltinExtensionApiWatcher('registries');
+
+    // get extensions folder
+    const extensionsFolder = resolve(__dirname, '../extensions/');
+
+    // loop on all subfolders from the extensions folder
+    const extensionsFolders = readdirSync(extensionsFolder, { withFileTypes: true })
+      .filter(dirent => dirent.isDirectory())
+      .map(dirent => resolve(__dirname, '../extensions/' + dirent.name));
+
+    // for each subfolder, start a watcher
+    for (const extensionFolder of extensionsFolders) {
+      setupExtensionApiWatcher(extensionFolder);
+    }
```

Comment:
nit: can be shorter and does not need to resolve extension folders twice
```suggestion
    // loop on all subfolders from the extensions folder
    readdirSync(extensionsFolder, { withFileTypes: true })
      .filter(dirent => dirent.isDirectory())
      .forEach(dirent => setupExtensionApiWatcher(join(extensionsFolder, dirent.name)));
```

Reply:
agree, I came from reworking two separate blocks so I didn't think to optimize